docs/manual/getting_started/installation/prerequisites.rst

```diff
@@ -0,0 +1,84 @@
+.. _prerequisites:
+
+.. links to external sites that open in a separate tab:
```

Comment:
I think opening in a separate tab is determined by the browser?

Reply:
From my observations on other websites, sometimes websites do open in other tabs by default (regardless middle click or left click). So maybe Sphinx has this feature as well?

But you are right, I don't see any option for it here. So it will probably follow browser defaults. Maybe:
```suggestion
.. links to external sites:
```

Reply:
This is just a comment for the functionality of:

.. |url::git_link| raw:: html

   <a href="https://git-scm.com/" target="_blank">Git</a>

The links are "asides", that is, these open in separate tabs so that the reader has them available without losing their place in the main document.